robolectric-shadows/shadows-core/src/main/java/org/robolectric/shadows/ShadowActivity.java

```diff
@@ -431,6 +431,16 @@ public IntentForResult(Intent intent, int requestCode, Bundle options) {
   }
 
   @Implementation
+  public void startActivities(Intent[] intents) {
```

Comment:
I think you can avoid having these boiler plate methods if you use

@Implements(inheritImplementationMethods = true) on the class?